src/modules/m_nick.c

```diff
@@ -332,7 +332,7 @@ void nick_identify (Nick *nptr, User *uptr __unused, char *all)
     NoticeToUser(nptr,"You are now identified");
     nptr->loginattempts = 0;
     nptr->lasttry = 0;
-    nptr->svid = user->nick;
+    strncpy(nptr->svid, user->nick, SVIDLEN + 1);
```

Comment:
Use SVIDLEN and not SVIDLEN + 1, otherwise the trailing '\0' could be overwritten.